flashlight/fl/nn/modules/Module.h

```diff
@@ -96,6 +106,15 @@ class Module {
    */
   virtual void setParams(const Variable& var, int position);
 
+  /**
+   * Copies the modules parameters, detaching from the computation graph.
+   *
+   * @param forceDeepCopy Forces a deep copy of the `Variable`s. See
+   * `Variable::copy` for more details about behaviour.
+   * @return a copy of the modules parameters as a vector of `Variable`
+   */
+  virtual std::vector<Variable> copyParams(bool forceDeepCopy = false) const;
```

Comment:
Might've left this in an earlier comment (apologies if I did; I can't find it) -- I think `forceDeepCopy` will be used infrequently enough to justify not including it as an option here. If users actually want to force an _eager_ deep copy, that's specialized behavior that they should implement themselves.

Reply:
Yep, above, its related to the `Variable::copy()` comment, I agreed to remove it.